app/src/main/java/space/taran/arkshelf/domain/UserPreferences.kt

```diff
@@ -0,0 +1,8 @@
+package space.taran.arkshelf.domain
+
+import java.nio.file.Path
+
+interface UserPreferences {
```

Comment:
Let's have single dedicated folder for saving links instead of asking user every time.
The folder should be configurable in settings menu.